### PR TITLE
docs: add advanced data source picker option in Explore

### PR DESCRIPTION
**Pull request by imatwawana**
References the advanced data source picker option in Explore.
Related to PR https://github.com/grafana/grafana/pull/70054 and issue https://github.com/grafana/grafana/issues/66916.

### CODE REVIEW

File: docs/sources/datasources/jaeger/_index.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ There are two ways to configure the trace to logs feature:
 #### Use a simple configuration
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/jaeger/_index.md:78]:
```suggestion
```

File: docs/sources/datasources/jaeger/_index.md [hunk 2]
```diff
@@ -127,6 +130,9 @@ The **Trace to metrics** setting configures the [trace to metrics feature](/blog
 To configure trace to metrics:
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/jaeger/_index.md:135]:
```suggestion
```

File: docs/sources/datasources/tempo/_index.md [hunk 3]
```diff
@@ -73,6 +73,9 @@ There are two ways to configure the trace to logs feature:
 #### Use a simple configuration
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/tempo/_index.md:78]:
```suggestion
```

File: docs/sources/datasources/tempo/_index.md [hunk 4]
```diff
@@ -127,6 +130,9 @@ The **Trace to metrics** setting configures the [trace to metrics feature](/blog
 To configure trace to metrics:
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/tempo/_index.md:135]:
```suggestion
```

File: docs/sources/datasources/zipkin/_index.md [hunk 5]
```diff
@@ -71,6 +71,9 @@ There are two ways to configure the trace to logs feature:
 #### Use a simple configuration
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/zipkin/_index.md:76]:
```suggestion
```

File: docs/sources/datasources/zipkin/_index.md [hunk 6]
```diff
@@ -125,6 +128,9 @@ The **Trace to metrics** setting configures the [trace to metrics feature](/blog
 To configure trace to metrics:
 
 1. Select the target data source.
+
+You can also click **Open advanced data source picker** to add a data source.
+
```
Comment (imatwawana) [inline docs/sources/datasources/zipkin/_index.md:133]:
```suggestion
```

**Comment by imatwawana** [comment]
Related to [PR 70054](https://github.com/grafana/grafana/pull/70054) and [issue 66916](https://github.com/grafana/grafana/issues/66916).

**Review by ivanortegaalba — APPROVED** [review]
Looks great to me! 💯

File: docs/sources/explore/_index.md [hunk 7]
```diff
@@ -41,9 +41,12 @@ To access Explore:
 
    Alternately to start with an existing query in a panel, choose the Explore option from the Panel menu. This opens an Explore tab with the query from the panel and allows you to tweak or iterate in the query outside of your dashboard.
 
-   {{< figure src="/static/img/docs/explore/panel_dropdown-7-4.png" class="docs-image--no-shadow" max-width= "650px" caption="Screenshot of the new Explore option in the panel menu" >}}
+   {{< figure src="/media/docs/grafana/panels-visualizations/screenshot-panel-menu-10.1.png" class="docs-image--no-shadow" max-width= "650px" caption="Screenshot of the panel menu including the Explore option" >}}
+
+1. Choose your data source from the drop-down in the top left.
+
+   You can also click **Open advanced data source picker** to see more data source options including adding a data source.
```
Comment (imatwawana) [inline docs/sources/explore/_index.md:48]:
```suggestion
   {{< figure src="/media/docs/grafana/panels-visualizations/screenshot-panel-menu-10.1.png" class="docs-image--no-shadow" max-width= "650px" caption="Screenshot of the panel menu including the Explore option" >}}

     1. Choose your data source from the drop-down in the top left.

   You can also click **Open advanced data source picker** to see more data source options including adding a data source.
```

File: docs/sources/explore/_index.md [hunk 8]
```diff
@@ -58,7 +61,7 @@ To open the split view:
 
 It is possible to select another data source for the new query which for example, allows you to compare the same query for two different servers or to compare the staging environment to the production environment.
 
-{{< figure src="/static/img/docs/explore/explore_split-7-4.png" max-width= "950px" caption="Screenshot of Explore option in the panel menu" >}}
+{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-explore-split-10.1.png" max-width= "950px" caption="Screenshot of Explore screen split" >}}
```
Comment (imatwawana) [inline docs/sources/explore/_index.md:64]:
```suggestion
{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-explore-split-10.1.png" max-width= "950px" >}}
```

**Pull request merged**